pkg/virt-controller/watch/migration.go

```diff
@@ -1679,6 +1739,10 @@ func (c *MigrationController) matchMigrationPolicy(vmi *virtv1.VirtualMachineIns
 	return nil
 }
 
+func (c *MigrationController) isMigrationHandedOffToVirtHandler(migration *virtv1.VirtualMachineInstanceMigration, vmi *virtv1.VirtualMachineInstance) bool {
```

Comment:
we don't need this function and the `isMigrationHandedOff` function. we just need one function to indicated if the hand off has occurred.

Reply:
Done :+1: